analyses/molecular-subtyping-HGG/results/HGG_defining_lesions.tsv

```diff
@@ -304,6 +303,7 @@ PT_F3S9PB94	7316-2240	BS_R1CV479M	No	No	No	No	No	No	FALSE
 PT_F822DBA0	7316-458	BS_KPWRDRFP	No	No	No	No	No	No	FALSE
 PT_F9F3E8YR	7316-3519	BS_R922P3XJ	No	No	No	No	No	No	FALSE
 PT_FAQWD7GX	7316-2311	BS_CN7PB0DN	No	No	No	No	No	No	FALSE
+PT_FCM7XPVR	7316-2285	BS_4GKH983E	No	No	No	No	No	No	FALSE
```

Comment:
The code generating HGG_defining_lesions.tsv is generated by 
- checking for histone variant status in all samples in the consensus maf [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/fbdb9a004fa667f431af0c983d0c4f71f4149939/analyses/molecular-subtyping-HGG/01-HGG-molecular-subtyping-defining-lesions.Rmd#L94-L137)
- removing all LGAT samples since LGG with Histone variant would be a subtype of LGG only
 
Since this sample was removed from LGG it is now being added to this file.